packages/pages/src/common/src/template/internal/getTemplateFilepaths.test.ts

```diff
@@ -0,0 +1,48 @@
+import path from "path";
+import { getTemplateFilepaths } from "./getTemplateFilepaths.js";
+import minimatch from "minimatch";
+import { Path } from "../../project/path.js";
+
+const rootPath = "src/templates";
+const domain1Path = "src/templates/some.domain1.com";
+const domain2Path = "src/templates/some.domain2.com";
+jest.mock("glob", () => {
+  return {
+    sync: (glob: string) => {
+      const filepaths = [
+        `${domain1Path}/brand1.tsx`,
+        `${domain1Path}/test.tsx`,
+        `${domain2Path}/brand2.tsx`,
+        `${domain2Path}/test.tsx`,
+        `${rootPath}/share.tsx`,
+        `${rootPath}/test.tsx`,
+      ];
+      return filepaths.filter((f) => minimatch(path.resolve(f), glob));
+    },
+  };
+});
+
+describe("getTemplateFilepaths", () => {
+  it("collects all template files from root folder path", () => {
```

Comment:
I'm not sure what the spec defines, but are we now treating subfolders as scopes? This will break existing usages if someone is using subfolders and they're no longer globbed.

Reply:
Oh interesting. I thought we were globbing everything in src/templates but it sounds like we're only globbing files at the root. If that's the case then your change isn't breaking.

Reply:
the logic in main is globbing everything inside src/templates with subfolders files too. But that also include folder paths (vs file paths) which is currently not being handle properly.

Reply:
[Confirmed with PM](https://yext.slack.com/archives/C02LLE9BW2K/p1666624017742579?thread_ts=1666623883.329669&cid=C02LLE9BW2K): we only use templates at the root level. The public documentation currently states to not put sub directories into the templates directory, so we are sticking with that paradigm for now

Reply:
Sounds good